packages/text-input/text-input.scss

```diff
@@ -253,8 +253,16 @@ $text-input-selection-color--inverted: color.scale(
 
 .jkl-text-area {
     height: auto;
+    min-width: rem(280px); // For å unngå "jank" med ekspanderende textarea med teller på små bredder
```

Comment:
Med denne kan man ikkje TextArea som er smalare enn 280px. Er tanken at dette er smalare enn man har bruk for i alle brukstilfeller?